cmd/juju/controller/showcontroller.go

```diff
@@ -168,6 +169,23 @@ func (c *showControllerCommand) Run(ctx *cmd.Context) error {
 			}
 		}
 
+		var mongoVersion string
+		// Check to see if we can call the mongo version. Older versions of the
+		// api do not support mongo version, so we should even prevent the call.
+		if apiVersion := client.BestAPIVersion(); apiVersion >= 6 {
+			mongoVersion, err = client.MongoVersion()
+			if err != nil {
+				details.Errors = append(details.Errors, err.Error())
+				continue
+			}
+		}
+
+		// Update client store.
+		if err := c.SetControllerModels(c.store, controllerName, allModels); err != nil {
```

Comment:
I think this block should be removed; it fails with a permission error if the user has no SuperuserAccess so I had that moved to L164 to keep things working for users with login (or with the add-model) permission.

Reply:
You're right it's a rebase issue.